### PR TITLE
Enable regression tests on Acorn

### PR DESCRIPTION
**Pull request by DusanJovic-NOAA**
# PR Checklist

- [ ] This PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [ ] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [ ] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [ ] Results for one or more of the regression tests change and the reasons for the changes are understood and explained below.

- [ ] New or updated input data is required by this PR. If checked, please work with the code managers to update input data sets on all platforms.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsibility to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

This PR updates rt scripts and adds configuration files required to run regression tests on Acorn.

Several tests need to be disabled due to various issues:

`datm_cdeps_debug_cfsr`, `cpld_debug_p8` and `cpld_debug_noaero_p8` crash with the following runtime error:

```
+ mpiexec -n 40 -ppn 40 -depth 1 ./fv3.exe
forrtl: error (182): floating invalid - possible uninitialized real/complex variable.
Image              PC                Routine            Line        Source            
fv3.exe            0000000008BB480B  Unknown               Unknown  Unknown
libpthread-2.31.s  000015198DCAF8C0  Unknown               Unknown  Unknown
fv3.exe            00000000061969D3  ice_comp_nuopc_mp         186  ice_comp_nuopc.F90
fv3.exe            0000000001C4A176  _ZN5ESMCI6FTable1        2167  ESMCI_FTable.C
fv3.exe            0000000001C4DDEA  ESMCI_FTableCallE         824  ESMCI_FTable.C
fv3.exe            000000000235178F  _ZN5ESMCI3VMK5ent        2308  ESMCI_VMKernel.C
fv3.exe            000000000233B139  _ZN5ESMCI2VM5ente        1216  ESMCI_VM.C
fv3.exe            0000000001C4B5F7  c_esmc_ftablecall         981  ESMCI_FTable.C
fv3.exe            000000000148FD6F  esmf_compmod_mp_e        1222  ESMF_Comp.F90

```
`fv3_regional_netcdf_parallel` fails while comparing baselines:
```
Traceback (most recent call last):
  File "/lfs/h1/emc/eib/noscrub/dusan.jovic/ufs/rt_on_acorn/ufs-weather-model/tests/compare_ncfile.py", line 6, in <module>
    with Dataset(sys.argv[1]) as nc1, Dataset(sys.argv[2]) as nc2:
  File "src/netCDF4/_netCDF4.pyx", line 2330, in netCDF4._netCDF4.Dataset.__init__
  File "src/netCDF4/_netCDF4.pyx", line 1948, in netCDF4._netCDF4._ensure_nc_success
OSError: [Errno -101] NetCDF: HDF error: b'/lfs/h1/emc/nems/noscrub/emc.nems/RT/NEMSfv3gfs/develop-20220623/INTEL/fv3_regional_netcdf_parallel/dynf000.nc'
```
`cpld_bmark_p8` and `cpld_restart_bmark_p8` require 11 nodes and just wait in the queue for hours.

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues must always be created before starting work on a PR branch!) 
- fixes #<issue_number>
- fixes noaa-emc/fv3atm/issues/<issue_number>

## Testing

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [x] hera.intel
- [x] hera.gnu
- [x] orion.intel
- [x] cheyenne.intel 
- [x] cheyenne.gnu
- [x] gaea.intel 
- [x] jet.intel
- [x] wcoss2.intel
- [x] acorn.intel
- [ ] opnReqTest for newly added/changed feature
- [ ] CI

## Dependencies

If testing this branch requires non-default branches in other repositories, list them. Those branches should have matching names (ideally).

Do PRs in upstream repositories need to be merged first?
If so add the "waiting for other repos" label and list the upstream PRs
- waiting on noaa-emc/nems/pull/<pr_number>
- waiting on noaa-emc/fv3atm/pull/<pr_number>

### CODE REVIEW

File: modulefiles/ufs_acorn.intel [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+module load PrgEnv-intel/8.1.0
+module load intel/19.1.3.304
+module load craype/2.7.13
+module load cray-mpich/8.1.7
+
+module load cmake/3.20.2
+
+module use /lfs/h1/emc/nceplibs/noscrub/hpc-stack/libs/hpc-stack/modulefiles/stack
+module load hpc/1.2.0
```
Comment (junwang-noaa) [inline modulefiles/ufs_acorn.intel:17]: It looks to me we are using hpc/1.1.0 on other platforms (hera,orion etc). Is there any difference between hpc/1.2.0 and hpc/1.1.0?

File: modulefiles/ufs_acorn.intel [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#%Module
+
+proc ModulesHelp {} {
+  puts stderr "\tcit - loads modules required for building and running UFS Model on Acorn"
+}
+
+module-whatis "loads UFS Model prerequisites for Acorn"
+
+module load PrgEnv-intel/8.1.0
+module load intel/19.1.3.304
```
Comment (junwang-noaa) [inline modulefiles/ufs_acorn.intel:10]: Can we have intel /2022 version?

File: tests/rt.conf [hunk 3]
```diff
@@ -13,18 +13,18 @@ RUN     | cpld_mpi_p8
 RUN     | cpld_control_c192_p8                                                                                                    | - wcoss2.intel jet.intel                | fv3 |
 RUN     | cpld_restart_c192_p8                                                                                                    | - wcoss2.intel jet.intel                |     | cpld_control_c192_p8
 
-RUN     | cpld_bmark_p8                                                                                                           | - wcoss2.intel jet.intel                | fv3 |
-RUN     | cpld_restart_bmark_p8                                                                                                   | - wcoss2.intel jet.intel                |     | cpld_bmark_p8
+RUN     | cpld_bmark_p8                                                                                                           | - wcoss2.intel acorn.intel jet.intel    | fv3 |
+RUN     | cpld_restart_bmark_p8                                                                                                   | - wcoss2.intel acorn.intel jet.intel    |     | cpld_bmark_p8
```
Comment (junwang-noaa) [inline tests/rt.conf]: What is the issue with cpld_bmark_p8, cpld_debug_p8 and cpld_debug_noaero_p8 on acorn?

File: modulefiles/ufs_acorn.intel (hunk 2 above)
Comment (DusanJovic-NOAA) [reply modulefiles/ufs_acorn.intel:10]: I don't know. At this moment this is the only compiler available.

File: modulefiles/ufs_acorn.intel (hunk 1 above)
Comment (DusanJovic-NOAA) [reply modulefiles/ufs_acorn.intel:17]: I'm not sure about the hpc module but all libraries are exactly the same versions as on the other platforms.

**Comment by jkbk2004** [comment]
@DusanJovic-NOAA we can start working on this PR. @DeniseWorthen @DavidHuber-NOAA Can we combine in #1299 and #1318 to this PR? With no baseline change, we can make a quick progress on tests.

**Comment by DavidHuber-NOAA** [comment]
@jkbk2004 Yes, I am OK with combining #1318 with this PR.

**Comment by DusanJovic-NOAA** [comment]
My branch is up to date with develop. @DavidHuber-NOAA @DeniseWorthen please open PRs to my branch.

**Comment by DusanJovic-NOAA** [comment]
#1299 and #1318 are merged into this branch. Ready for regression testing.

**Comment by jkbk2004** [comment]
> #1299 and #1318 are merged into this branch. Ready for regression testing.

@DusanJovic-NOAA thanks! I will start from Cheyenne. Hopefully will catch up orion and jet before they start maintenance.

**Comment by BrianCurtis-NOAA** [comment]
@DusanJovic-NOAA LIne 197 in rt.conf the HAFS-ALL compile. Please block wcoss2.intel for that compile and those tests that follow. We have to wait for NCO to install pio/2.5.3 on WCOSS2 before those are run.

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: gaea
Compiler: intel
Job: RT
[RT] Repo location: /lustre/f2/pdata/ncep/emc.nemspara/autort/pr/985259239/20220726190007/ufs-weather-model
[RT] Error: Test cpld_control_c192_p8 007 failed in run_test failed 
[RT] Error: Test compile_011 failed in run_compile failed 
Please make changes and add the following label back: gaea-intel-RT

**Comment by BrianCurtis-NOAA** [comment]
@DusanJovic-NOAA Could you edit the PR template to include Acorn.intel ?

**Comment by DusanJovic-NOAA** [comment]
> @DusanJovic-NOAA Could you edit the PR template to include Acorn.intel ?

I already added acorn.intel to the pr template. See https://github.com/ufs-community/ufs-weather-model/pull/1307/commits/14821436e19a9a79912a8dbda74d7f9a29ff8732

**Comment by DeniseWorthen** [comment]
CMEPS PR has been merged. New hash is [f9f7541](https://github.com/NOAA-EMC/CMEPS/commit/f9f7541a4a7e7fd6009a3029c883bbbb289d6ef4)

**Review by BrianCurtis-NOAA — APPROVED** [review]

**Comment by jkbk2004** [comment]
@DusanJovic-NOAA can you resolve the conversations? @binli2337 @junwang-noaa it looks the pr is ready to merge. Can you take a look and approve?

**Review by junwang-noaa — APPROVED** [review]

**Comment by DavidHuber-NOAA** [comment]
All regression tests passed on S4.

**Pull request merged**